shared/featureconfig/flags.go

```diff
@@ -106,9 +106,9 @@ var (
 		Name:  "enable-next-slot-state-cache",
 		Usage: "Improves attesting and proposing efficiency by caching the next slot state at the end of the current slot",
 	}
-	updateHeadTimely = &cli.BoolFlag{
-		Name:  "update-head-timely",
-		Usage: "Improves update head time by updating head right after state transition",
+	disableUpdateHeadTimely = &cli.BoolFlag{
+		Name:  "disable-update-head-timely",
+		Usage: "Disables updating head time right after state transition",
```

Comment:
```suggestion
		Usage: "Disables updating head right after state transition",
```
Not 100% sure about this suggestion.